<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabWidget</class>
 <widget class="QTabWidget" name="TabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Corona Statistics</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tab_single_country">
   <attribute name="title">
    <string>Single Country</string>
   </attribute>
   <widget class="QCheckBox" name="chk_daily_inc_log1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>570</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Daily Increase Log</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_confirmed_cases_log1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>550</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirmed Cases Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_plot1">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>570</y>
      <width>99</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_select">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Country Selection: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_options1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>530</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Options:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lst_country11">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>321</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_compare_countries">
   <attribute name="title">
    <string>Compare Countries</string>
   </attribute>
   <widget class="QCheckBox" name="chk_confirmed_cases_log2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>560</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirmed Cases Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_plot2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>560</y>
      <width>99</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_country21">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>First Country Selection: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_country22">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Second Country Selection: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_options2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>540</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Options:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lst_country21">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>321</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lst_country22">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>321</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_viz">
   <attribute name="title">
    <string>Visualization</string>
   </attribute>
   <widget class="QSlider" name="sld_min">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>861</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sld_max">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>580</y>
      <width>861</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_options3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Options:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
